lilypad/generations.py

```diff
@@ -386,8 +400,10 @@ def inner(*args: _P.args, **kwargs: _P.kwargs) -> _R:
                     output = fn(*args, **kwargs)
                     if isinstance(output, BaseModel):
                         output = str(output.model_dump())
+                    # Capture trace/span ID
+                    trace_id = format(span.get_span_context().trace_id, "x")
```

Comment:
@willbakst 
Is this what you mean by trace_id in your comment?
However, it may not be possible to get it at this pre-commit stage, so you may need to delay and get it in the Genration class.
And as for the span_id, are you talking about the DB or the ID of the span here?

Reply:
I checked at hand and found that trace_id and span_id could be obtained in this way, so add span_id as well.